src/app/components/hris/employees/employee-profile/accordions/accordion-administrative-documents/accordion-administrative-documents.component.ts

```diff
@@ -101,7 +101,7 @@ export class AccordionAdministrativeDocumentsComponent {
     if (this.allowedTypes.includes(this.selectedFile.type)) {
```

Comment:
You're going to have to add another conditional check to determine if the uploadButtonIndex is the index of Written Approvals and if the allowed types are included. Else if only PDF is included. Else throw the snackbar message.

Reply:
Made use of a conditional check